pkg/tsdb/grafana-pyroscope-datasource/phlareClient_test.go

```diff
@@ -1,110 +0,0 @@
-package phlare
```

Comment:
Why delete this whole file.. are the tests covered elsewhere?

Reply:
🤦 no deleted by accident, thanks for catching this.